### PR TITLE
Refactoring a benchmark tool

### PR DESCRIPTION
**Pull request by hanbitmyths**
- Refactor index.js into multiple typescript files
- Support synthetic input data generation referring to model and a hint from configuration when an input is dynamic
- Make ORT Web SIMD configurable
- Make Tensorflow.js threading and SIMD configurable
- Replace a sample model from super resolution to facemesh

### CODE REVIEW

File: sample-configs/run_config_mediapipe_facemesh.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "benchmarkName": "mediapipe facemesh",
+    "frameworksToTest": ["tensorflow.js", "ort-web"],
+    "backendsToTest": ["wasm", "webgl"],
+    "warmupIteration": 5,
+    "runIteration": 25,
+    "logLevel": "error",
+    "debug": false,
+    "ortweb": {
```
Comment (duli2012) [inline sample-configs/run_config_mediapipe_facemesh.json:9]: don't see ort shape info here. will it share the information with tfjs config?

File: sample-configs/run_config_softmax_test.json [hunk 2]
```diff
@@ -1,30 +1,12 @@
 {
```
Comment (duli2012) [inline sample-configs/run_config_softmax_test.json:1]: No input shape for either ort web or tfjs?

**Review by duli2012 — APPROVED** [review]

File: sample-configs/run_config_mediapipe_facemesh.json (hunk 1 above)
Comment (hanbitmyths) [reply sample-configs/run_config_mediapipe_facemesh.json:9]: This model has a shape with fixed dimensions. In this case, a tool reads input shape information from a onnx model and a tfjs model automatically.

File: sample-configs/run_config_softmax_test.json (hunk 2 above)
Comment (hanbitmyths) [reply sample-configs/run_config_softmax_test.json:1]: Same as the above.

File: sample-configs/run_config_mediapipe_facemesh.json (hunk 1 above)
Comment (hanbitmyths) [reply sample-configs/run_config_mediapipe_facemesh.json:9]: tfjs shape is not required, either. But, I intentionally put a shape for an example.

Comment (duli2012) [reply sample-configs/run_config_mediapipe_facemesh.json:9]: thanks for clarification.

**Pull request merged**